packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +98,12 @@ export const Sidebar = () => {
         (updateStatusSuite !== 'up-to-date' && !closedNotificationSuite) ||
         (!['up-to-date', 'disconnected'].includes(updateStatusDevice) && !closedNotificationDevice);
 
+    const showAccountsWithExtra =
+        !shouldDisplayDeviceCompromised &&
+        selectedDevice !== undefined &&
+        selectedDevice.mode !== 'normal' && // bootloader, etc...
```

Comment:
I think you meant this?
```suggestion
        selectedDevice.mode === 'normal' && // bootloader, etc...
```

Reply:
yes, its there? and it was there `===` yesterday.. it seems I have maybe forget co push or something :cry: